libraries/localization-utilities/src/TypingsGenerator.ts

```diff
@@ -33,17 +52,24 @@ export interface ITypingsGeneratorOptions extends ITypingsGeneratorBaseOptions {
  */
 export class TypingsGenerator extends StringValuesTypingsGenerator {
   public constructor(options: ITypingsGeneratorOptions) {
-    const { ignoreString, processComment } = options;
+    const {
+      ignoreString,
+      processComment,
+      terminal,
+      resxNewlineNormalization,
+      ignoreMissingResxComments,
+      inferDefaultExportInterfaceNameFromFilename
+    } = options;
     super({
       ...options,
       fileExtensions: ['.resx', '.resx.json', '.loc.json', '.resjson'],
       parseAndGenerateTypings: (fileContents: string, filePath: string, resxFilePath: string) => {
         const locFileData: ILocalizationFile = parseLocFile({
-          filePath: filePath,
+          filePath,
           content: fileContents,
-          terminal: this._options.terminal!,
-          resxNewlineNormalization: options.resxNewlineNormalization,
-          ignoreMissingResxComments: options.ignoreMissingResxComments,
+          terminal: terminal!,
```

Comment:
Why does this need the non-null assertion? Could we change the interface to make it required?